src/Core/WinSWCore/LogAppenders.cs

```diff
@@ -479,6 +483,30 @@ private void ZipFiles(string directory, string fileExtension, string zipFileBase
             {
                 EventLogger.LogEvent($"Failed to Zip files. Error {e.Message}");
             }
+
+            if(!(ZipDaysToKeep is null) && ZipDaysToKeep != 0)
+            {
+                try
+                {
+                    List<string> ZipFiles = new List<string>(Directory.GetFiles(directory, zipFileBaseName + ".*.zip"));

```

Comment:
This pattern will remove zip files by other mode.

Reply:
Code has been changed to only delete zip files that match the configured pattern.